docs/template/public/less/_kss-content.less

```diff
@@ -0,0 +1,206 @@
+// shared styles across pages 
+#kss-content {
+    .sectionSpacing {
+        padding-bottom: 30px;
+        padding-top: 30px;
+    }
+    .table-behavoir-cell .seperator {
+        margin: 0 50px 0 0;
+    }
+
+    /* patterns */
+    .row.visual-code {
+        margin: 0;
+    }
+
+    h3.popover-title {
+        margin-bottom: 0;
+    }
+
+    .usercard-info p {
+        margin-bottom: 0;
+    }
+
+    .list-group-item.deleteable .list-group-item-delete {
+        top: 18%;
+    }
+
+    .row {
+        margin-right: -8px;
```

Comment:
this doesn't look to be correct. we shouldn't need to compensate fro the bootstrap grid here